AdminServer/appscale/admin/instance_manager/instance.py

```diff
@@ -92,6 +92,20 @@ def create_java_start_cmd(app_name, port, load_balancer_host, max_heap,
   revision_base = os.path.join(UNPACK_ROOT, revision_key)
   web_inf_directory = find_web_inf(revision_base)
 
+  api_server = os.path.join(SCRIPTS_DIR, 'appscale_java8_apiserver.sh')
+  apis_using_external_server = ['app_identity_service', 'datastore_v3',
+                                'memcache', 'taskqueue']
+  api_server_flags = [
+    '--application={}'.format(app_name),
+    '--datastore_path={}'.format(
+      ':'.join([options.db_proxy, str(DB_SERVER_PORT)])),
+    '--login_server={}'.format(load_balancer_host),
```

Comment:
Is load_balancer a single host? Does this point to login_host or to one of the internal load balancers?

Reply:
`load_balancer_host` is the login property in this case.